### ISSUE TITLE
object has no attribute '_formatter_parser'

### ISSUE THREAD

**Issue by buddharajuramaraju** [RasaHQ/rasa_nlu#631]
**Rasa NLU version**  :"version": "0.10.1"

**Used backend / pipeline**  `spacy_sklearn`,

**Operating system** RHEL 7

**Issue**:
while executing query following error is comming

{"error": "'NoneType' object has no attribute '_formatter_parser'"}

Example:
 curl -XPOST localhost:5000/parse -d '{"q":"I am looking for Chinese food"}'
{"error": "'NoneType' object has no attribute '_formatter_parser'"}

**Content of configuration file** (if used & relevant):
```json

```

**Comment by wrathagom** [comment]
Can you provide more information on how you're running Rasa: command line, python, docker, etc?

Have you already trained a model? What is in the `/projects` folder? Can you post the metadata.json file of the model that is being loaded.

**Comment by derde** [comment]
I have the same on a brand new installation on Ubuntu from github: command line, docker, following the tutorial without modification.

```
{
    "entity_synonyms": "entity_synonyms.json", 
    "pipeline": [
        "nlp_spacy", 
        "tokenizer_spacy", 
        "intent_featurizer_spacy", 
        "intent_entity_featurizer_regex", 
        "ner_crf", 
        "ner_synonyms", 
        "intent_classifier_sklearn"
    ], 
    "spacy_model_name": "en", 
    "language": "en", 
    "intent_classifier_sklearn": "intent_classifier.pkl", 
    "trained_at": "20171020-125436", 
    "training_data": "training_data.json", 
    "entity_extractor_crf": {
        "BILOU_flag": true, 
        "model_file": "crf_model.pkl", 
        "version": 1, 
        "crf_features": [
            [
                "low", 
                "title", 
                "upper", 
                "pos", 
                "pos2"
            ], 
            [
                "bias", 
                "low", 
                "word3", 
                "word2", 
                "upper", 
                "title", 
                "digit", 
                "pos", 
                "pos2", 
                "pattern"
            ], 
            [
                "low", 
                "title", 
                "upper", 
                "pos", 
                "pos2"
            ]
        ]
    }, 
    "rasa_nlu_version": "0.10.1", 
    "regex_featurizer": "regex_featurizer.json"
}
```

**Comment by tmbo** [comment]
which python version do you use?

**Comment by tmbo** [comment]
And can you please also add the stacktrace from the console

**Comment by Jonathan-Wei** [comment]
Hi,@wrathagom,I have the same problem.
This is my metadata.json.
`{
    "entity_synonyms": "entity_synonyms.json",
    "pipeline": [
        "nlp_mitie",
        "tokenizer_jieba",
        "ner_mitie",
        "ner_synonyms",
        "intent_entity_featurizer_regex",
        "intent_featurizer_mitie",
        "intent_classifier_sklearn"
    ],
    "mitie_feature_extractor_fingerprint": 1810187658478185215,
    "regex_featurizer": null,
    "language": "zh",
    "mitie_file": "./data/total_word_feature_extractor_zh.dat",
    "intent_classifier_sklearn": "intent_classifier.pkl",
    "trained_at": "20171030-153135",
    "training_data": "training_data.json",
    "entity_extractor_mitie": "entity_extractor.dat",
    "rasa_nlu_version": "0.10.1"
}`

**Comment by Jonathan-Wei** [comment]
@wrathagom @tmbo python version is 2.7.0

**Comment by derde** [comment]
I pushed a fix/work-around (it's a log statement that doesn't have a format string), so if you get the current version of rasa_nlu you should find the visible part of this problem has gone away.

**Comment by tmbo** [comment]
@Jonathan-Wei we also just recently cut a release, so the issue is also fixed in `0.10.4`
